keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -575,13 +624,14 @@ enabled even if the annotation has been set on the Service.
   EndpointSlices for Services that have this feature enabled.
 
 * **Are there any tests for feature enablement/disablement?**
-  Per Service enablement and disablement is covered in depth by unit tests. As a
-  prerequisite for graduation to GA, we will also add the following:
-
-  - Test coverage in EndpointSlice strategy to ensure that the Hints field is
-    dropped when the feature gate is not enabled.
-  - Test coverage in EndpointSlice controller for the transition from enabled to
-    disabled.
+  Enablement is covered by a variety of tests:
+
+  * Per Service enablement and disablement in EndpointSlice Controller. [(Unit
+    Tests.)](https://github.com/kubernetes/kubernetes/blob/468ce5918377ab4d4e3180b4fd33fdd2bdb16ec9/pkg/controller/endpointslice/reconciler_test.go#L1641-L1907)
+  * Hints field is dropped when feature gate is off. [(Strategy Unit
+    Tests.)](https://github.com/kubernetes/kubernetes/blob/468ce5918377ab4d4e3180b4fd33fdd2bdb16ec9/pkg/registry/discovery/endpointslice/strategy_test.go)
+  * TODO before GA: Test coverage in EndpointSlice controller for the transition
```

Comment:
@robscott  - FWIW - it should have happened for beta, as this is when most users generally enable it (beta on by default).
Please keep that in mind for the future (and I learnt that over time and now kind-of require it as beta criteria in many cases).